### ISSUE TITLE
Information share with developer's multiplayer server

### ISSUE THREAD

**Issue by Protektor-Desura** [desura/Desurium#140]
We need a way to cause the client to send the user's information to the developer's server so an account on the developer's server can be setup for multiplayer games that require this option.

There is already a URL for each game that if developers set it up users can opt-in to send their information to the developer. Basically this would be a forced call to that URL for each game. Basically the game won't launch until you do it, the same way the EULA is handled.

It would need some kind of pop-up like the EULA saying you are going to be sending this info and it is needed for the game to function properly. They can always decline but then the game isn't useful so it shouldn't launch just like not agreeing to the EULA.

There are already several games on the system that require this and we give out CDKeys when you buy it to, then the users enters it on the dev's website to setup an account on their servers. It isn't the most practical or pain free way to set things up. 

A more automated method like this would be better for the users and the developers both.

**Comment by Jookia** [comment]
So it'd be an API call?

**Comment by karolherbst** [comment]
We discussed this in IRC:

short:
desura must have the ability to send some information with the API call so the game server can create an account for multiplayer.
before that a message/pop-up will appear so the user has the ability to decide this

I hope I don't mix up something

**Comment by Protektor-Desura** [comment]
That is correct but it shouldn't really be a big feature. There is already a URL for each game to send the user's information to the developer. So all that would have to happen is that when the client gets the bit/request that the game needs it, then it would pop-up a confirm window then call that URL. So it shouldn't be too much more complicated than the List games & Keys was to add. We only need the server to have an option in the game profile that developers can check to send the request to the client. If the developer has it setup then it is the "Connect" tab on the game's profile page.
